target/linux/ramips/image/mt7621.mk

```diff
@@ -9,6 +9,31 @@ DEFAULT_SOC := mt7621
 KERNEL_DTB += -d21
 DEVICE_VARS += ELECOM_HWNAME LINKSYS_HWNAME
 
+define Build/beeline-sb-flash-header
+	echo -ne "$$(echo 000001001c0000000000420000000000 | sed \
+		's/../\\x&/g')" | dd of=$@.tmp bs=1 count=16 \
+		conv=notrunc status=none 2>/dev/null
```

Comment:
I think `status=none` can be skipped.

Reply:
All "status=none" occurrences were removed from the recipe. All is ok. Thanks.